add-ons/tb68/bootstrapped.md

```diff
@@ -60,3 +60,65 @@ The shown example also specifies an optional `options` key to define the options
 This example is only in English. You probably want to use translated strings in your manifest. Read [this MDN article about it](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Internationalizing_manifest.json). Unfortunately that means you now need two sets of translated strings, one \(that you already have\) for your extension and another for the manifest.
 {% endhint %}
 
+## Update handling of windows on startup
+
+Prior to Thunderbird 68, the `startup()` function for bootstrapped extensions was called after windows were fully loaded. Therefore, it was common for bootstrapped extensions to assume that document elements they wanted to modify or augment already existed. However, bootstrapped extensions can no longer assume that the document is fully loaded when they start up.
+
+Here's an example of how to deal with this.
+
+```javascript
+const {Services} = ChromeUtils.import("resource://gre/modules/Services.jsm");
+
+var allInOneObserver = {
```

Comment:
Can we make the examples here adhere to mail's eslint config?

Reply:
I will try, but it would be nice if people who wanted to contribute docs to this repository did not have to clone the 320MB Thunderbird source tree and install the umpteen gazillion node modules needed to use eslint and the config in comm/.eslintrc.js.